### ISSUE TITLE
Discussion on plotting for research versus plotting for data journalism

### ISSUE THREAD

**Issue by joshwlambert** [epiverse-trace/tracetheme#5]
This issue is to discuss how Epiverse-TRACE should style their plots in terms of traditional academic data visualisation, versus the style commonly used in data journalism.

To me, the difference between plotting for research (mostly academic papers) and data journalism, is the former creates figures that do not makes conclusions in themselves. The figure is a general presentation of the data/results, and it is the text, and sometimes the figure legend, that gives the interpretation. Data journalism on the other hand, much more frequently annotates and highlights parts of plots in order to lead the reader directly to a conclusion, independent on supplementary text. For example the first figure in the [BBC's plotting style guide](https://bbc.github.io/rcookbook/). 

We should be consistent with how we decide to plot information within Epiverse-TRACE, accounting for differents between different output types. 

Some general points to discuss:
- What are peoples preference on the two styles of data visualisation?
- Should different Epiverse outputs have different styles of data visualisation? E.g. vignettes can be informal and use more of a data journalist style, while training material is more research-like.
- Should plotting methods for data types (e.g. S3 classes) be general enough to allow user to customise with annotations when needed and by included by default?
- Should common annotations be provided as `geoms`.

**Comment by chartgerink** [comment]
Good point @joshwlambert 👍 To me this would depend on the purpose of this specific library (I cannot find an explicitly documented purpose) and the frictions between the different target audiences. 

I understand the purpose of `tracetheme` to be **harmonize plotting outputs across packages** - I would take that as providing the flexibility for packages to decide how opinionated their plotting functions will be. 

A package targeted more towards policy, may provide less flexibility and more opinionated plotting to be fast + informative. 

Vice versa, a package targeted more towards researchers may provide more flexibility and fewer opinions in plotting, but will take longer to set up and will be more variable in how informative it is.

In other words, I would propose to have `tracetheme` provide no more opinionated theming than **what is provided in the packages**. This defers those decisions to the packages with their own respective target audiences and purposes (a form of [subsidiarity,](https://en.wikipedia.org/wiki/Subsidiarity) if you will).
